app/com/arpnetworking/kairos/client/KairosDBRequestException.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2019 Dropbox Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.kairos.client;
+
+import akka.http.javadsl.model.Uri;
+
+/**
+ * An exception that represents a non-2xx KairosDB request.
+ *
+ * @author Gilligan Markham (gmarkham at dropbox dot com)
+ */
+public class KairosDBRequestException extends RuntimeException {
```

Comment:
nit: should be `Db` not `DB`; I believe the existing test class (at least) is also wrong.